content/BCPatterns/generic-method-pattern/index.md

```diff
@@ -0,0 +1,327 @@
++++
+title = "Generic Method Pattern"
+weight = 1180
++++
+
+_Created by Gary Winter (Cloud Ready Software), Described by waldo (iFacto Business Solutions | Dynex)_
+
+## Abstract
+
+The goal of this pattern is to facilitate a lot of things in one single awesome way of writing code.  If you apply this apply this pattern as a general pattern, you'll implement:
+- Extensibility
+- Decouplability
```

Comment:
I think the better wording is "Decoupling" or "Loose coupling"

Reply:
Agree with @ajkauffmann